README.md

```diff
@@ -1,3 +1,18 @@
 # lyant - Learn You A Next Thing
 
 Bored? Want to learn something?
+
+Added github login, to run the project just open `app->controller->app.go`
```

Comment:
If you update your fork with the latest from this repository, you'll pick up `setup.py`.  You could look into copying what I've got there for the neo4j configuration and creating a method for the github secrets too.